tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1255,9 +1255,7 @@ pfc_asym/test_pfc_asym.py:
 
 pfc_asym/test_pfc_asym.py::test_pfc_asym_off_rx_pause_frames:
    skip:
-     reason: "skipped for Barefoot platform"
-     conditions:
-        - "asic_type in ['barefoot']"
```

Comment:
hi @byu343 could you clarify why skip all? the original skip is for barefoot only.

Reply:
Hi @StormLiangMS, pfc_asym/test_pfc_asym.py::test_pfc_asym_off_rx_pause_frames was skipped on non-barefoot platforms,  before the recent change of enabling longest match on conditions marks. 

The reason for that is: without the longest match, the rule of pfc_asym/test_pfc_asym.py in https://github.com/sonic-net/sonic-mgmt/pull/15246/files#diff-ab3274ff0521b1296970af58aea75edf72472ce5a2b29506f54c9ef0534fe1f2R1250 was chosen over the rule of pfc_asym/test_pfc_asym.py::test_pfc_asym_off_rx_pause_frames here for non-barefoot platforms, so pfc_asym/test_pfc_asym.py::test_pfc_asym_off_rx_pause_frames was skipped on non-barefoot platforms.

Assuming the earlier pfc_asym/test_pfc_asym.py::test_pfc_asym_off_rx_pause_frames skipped on non-barefoot platforms is as desired, here we could set it to skip on all platforms.